### PR TITLE
Create movie class

### LINKED ISSUE #9: [1.5pt] Create Movie class
- Create Movie class in a separate .rb file.

### PR DESCRIPTION
**Pull request by blase147**
## In this pull request, I implemented the following: ⬇️

- Created a Movie class in a separate .rb file. ✔️
- All Movie class properties visible in the diagram were defined and set up in the constructor method.. ✔️

### CODE REVIEW

File: movie.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require_relative './item'
+
+class Movie < Item
+  attr_reader true
```
Comment (SelmaNdapanda) [inline movie.rb]: You forgot to add a ":" sign to your attribute reader

File: movie.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+  def initialize
+    super(true)
+    @silent = true
```
Comment (SelmaNdapanda) [inline movie.rb]: I suggest you assign the initial value "true" and the silent variable as arguments to the constructor.
Like this: def initialize(silent: true)

**Review by SelmaNdapanda — CHANGES_REQUESTED** [review]
Hi @blase147 ,

Good job :+1:  at implementing the required changes,
however, there are some issues you need to fix to go to the next project.
Kindly check the inline comments under the review :arrow_down:

File: movie.rb (hunk 1 above)
Comment (blase147) [reply movie.rb]: Thanks for your feedback @SelmaNdapanda . I actually added that, but it was removed by rubocop. Do you think it must really be present?

Comment (SelmaNdapanda) [reply movie.rb]: You were supposed to put the variable silent instead of the initial value.
e.g attr_reader :silent

**Review by SelmaNdapanda — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @blase147 ,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

**Pull request merged**